src/Leela.cpp

```diff
@@ -277,6 +277,8 @@ static void parse_commandline(int argc, char *argv[]) {
             cfg_timemanage = TimeManagement::OFF;
         } else if (tm == "fast") {
             cfg_timemanage = TimeManagement::FAST;
+        } else if (tm == "simple") {
+            cfg_timemanage = TimeManagement::SIMPLE;
```

Comment:
We already make quite a few things conditional on noise/random selection so this could as well.

I understand that above you've added a possibility for the server to add arbitrary options to enable this? That has the disadvantage that it requires all clients to be switched. (Unless AutoGTP behaves correctly with unknown options).

Reply:
I thought that server can give different commands to different versions. Would you rather have it default to simple?

I think that autogtp ignores unknown options. It parses json to ```map<QString, QString>``` and only checks keys for commands it expects.

Reply:
>I thought that server can give different commands to different versions. 

Ha, good point.

>Would you rather have it default to simple?

We could do that if we detect noise is enabled. BTW. What about DONT_PRUNE or NO_PRUNING instead of SIMPLE?

Maybe FAST should have been named VERY_EARLY_EXIT or something. The naming here kinda sucks right now.